docs/menu.yml

```
@@ -171,8 +173,8 @@ catalog:
             path: "/en/setup/service-agent/browser-agent"
           - name: "Database Monitoring"
             catalog:
-              - name: "Virtual Database"
-                path: "/en/setup/service-agent/virtual-database"
+              - name: "MYSQL Monitoring"
```

```suggestion
              - name: "MySQL Monitoring"
```

Please check all `MySQL` in the doc, the upper and lower cases are defined by MySQL community, this is an official name, don't write it randomly

---

OK, I will change it later.